### PR TITLE
0.31 changelog

### PR DESCRIPTION
**Pull request by Bonapara**

### CODE REVIEW

**Comment by charlesBochet** [comment]
@Bonapara why are we changing the yarn builder version, would be great to upgrade it but have you tested it works on all projects? (we would need to delete the project, clone and install with yarn again)

**Comment by Bonapara** [comment]
@charlesBochet thought it was just a local upgrade for me. I switched back to 4.4.0. Is it fully reverted?

File: package.json [hunk 1]
```diff
@@ -340,7 +340,7 @@
   },
   "license": "AGPL-3.0",
   "name": "twenty",
-  "packageManager": "yarn@4.3.1",
+  "packageManager": "yarn@4.4.0",
```
Comment (charlesBochet) [inline package.json:343]: nope nope :p

Comment (charlesBochet) [reply package.json:343]: I think we should upgrade at some point but we should use the occasion to upgrade node at the same time :)
Let's keep it as it was

**Review by charlesBochet — APPROVED** [review]
LGTM!

**Pull request merged**